components/pd_client/src/lib.rs

```diff
@@ -149,6 +164,36 @@ impl BucketStat {
             *bytes += key.len() as u64 + value_size;
         }
     }
+
+    pub fn split(&mut self, idx: usize) {
```

Comment:
@bufferflies and I discussed previously that both split buckets should inherit the stats of the original bucket. Should we do this here as well?

Reply:
ok.